README.md

```diff
@@ -1,2 +1,187 @@
-# Math-magicians
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  Make simple calculations.  Read a random math-related quote.  By building this application, you will learn how to use React.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MATH MAGICIANS] <a name="about-project"></a>
+
+**[MATH MAGICIANS]** This project is called Math-magicians. It is a wep app for performing basic math operations while generating random math quotes.
+
+## 🛠 Built With <a name="HTML, CSS and Javascript"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features 
+- **[Project was created with React]**
+- **[Quotes was fetched from API]**
+- **[Pages are routed with react router]**
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="#"></a>
+
+
+- [[Live Demo Link](#)]
```

Comment:
- [ ] Your web app is not deployed kindly make sure to remove this `Live demo` section altogether from the README file